src/core/vaults/RMVault.sol

```diff
@@ -23,6 +23,12 @@ contract RMVault is RVaultBase, IManagedVault {
     /// @dev Version of RMVault implementation
     string public constant VERSION = '1.0.0';
 
+    /// @inheritdoc IVault
+    uint public constant UNIQUE_INIT_ADDRESSES = 1;
+
+    /// @inheritdoc IVault
+    uint public constant UNIQUE_INIT_NUMS = 0;
```

Comment:
use internal constant and public getter with overriding of vault base empty getter